### DISCUSSION TITLE
REST Query Dynamic variable and Map component

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#9335] [Help]
Hello,

Could someone help me ?
I have a REST Query with this result :
![image](https://user-images.githubusercontent.com/122595047/212256349-47f956e2-de81-4ac5-9948-0a27ae7fb16a.png)

I have set 2 dynamic variables based on positiony and positionx
![image](https://user-images.githubusercontent.com/122595047/212256579-63ee9d05-f0aa-4158-af9e-c84ed38986d5.png)

But when  I set the map component I cant select the 2 dynamic variables in Latitude Key and Longitude Key

**Comment by ghost** [comment]
Should i write {{ divide data.0.[positiony] 1000000 }} ? ... i've try, but it seems not working.

**Comment by melohagan** [comment]
Hey @Jules-seluJ 

Apologies for the late response!

You cannot use handlebars within dynamic variables, you can only reference the variable, e.g. `{{ data.0.[positiony] }}`

This is fine though, because you can update the transformer of your REST query to return the latitude and longitude in the format you want.

Something like:

```js
return data.map(element => {
  let updated = {...element}
  updated["positionx"] = updated["positionx"]/1000000
  updated["positiony"] = updated["positiony"]/1000000
  return updated
})
```
